static/app/views/issueDetails/streamline/hooks/useCopyIssueDetails.spec.tsx

```diff
@@ -0,0 +1,364 @@
+import {EventFixture} from 'sentry-fixture/event';
+import {GroupFixture} from 'sentry-fixture/group';
+
+import {renderHook} from 'sentry-test/reactTestingLibrary';
+
+import * as indicators from 'sentry/actionCreators/indicator';
+import {
+  type AutofixData,
+  AutofixStatus,
+  AutofixStepType,
+} from 'sentry/components/events/autofix/types';
+import * as autofixHooks from 'sentry/components/events/autofix/useAutofix';
+import type {GroupSummaryData} from 'sentry/components/group/groupSummary';
+import * as groupSummaryHooks from 'sentry/components/group/groupSummary';
+import type {EventTag} from 'sentry/types/event';
+import {useCopyIssueDetails} from 'sentry/views/issueDetails/streamline/hooks/useCopyIssueDetails';
+
+// Mock the internal helper function since it's not exported
+jest.mock('sentry/views/issueDetails/streamline/hooks/useCopyIssueDetails', () => {
+  // Store the original implementation
+  const originalModule = jest.requireActual(
+    'sentry/views/issueDetails/streamline/hooks/useCopyIssueDetails'
+  );
+
+  // Mock the internal function for our tests
+  const mockIssueAndEventToMarkdown = jest.fn(
```

Comment:
does this need to be mocked like this? Doesn't make sense to reimplement the entire thing